assets/webpack.config.js

```diff
@@ -8,40 +8,112 @@ const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
 const { sentryWebpackPlugin } = require("@sentry/webpack-plugin");
 
+const common_export_body = {
+  resolve: {
+    extensions: [".ts", ".tsx", ".js", ".jsx"],
+    alias: {
+      // Please also update the "paths" list in tsconfig.json when you add aliases here!
+      Components: path.resolve(__dirname, "src/components"),
+      Hooks: path.resolve(__dirname, "src/hooks"),
+      Util: path.resolve(__dirname, "src/util"),
+      Constants: path.resolve(__dirname, "src/constants"),
+      Images: path.resolve(__dirname, "static/images"),
+    },
+  },
+  output: {
+    filename: "[name].js",
+    path: path.resolve(__dirname, "../priv/static/js"),
+  },
+  devtool: "source-map",
+  optimization: {
+    minimizer: [new TerserPlugin(), new OptimizeCSSAssetsPlugin()],
+  },
+};
+
+function getCommonRules(isOfmPackage) {
+  return [
+    {
+      enforce: "pre",
+      test: /\.js$/,
+      loader: "source-map-loader",
+    },
+    {
+      test: /\.s?css$/,
+      use: [
+        MiniCssExtractPlugin.loader,
+        {
+          loader: "css-loader",
+        },
+        {
+          loader: "sass-loader",
+        },
+      ],
+    },
+    {
+      test: /\.svg$/i,
+      issuer: /\.[jt]sx?$/,
+      use: ["@svgr/webpack"],
+    },
+    {
+      test: /\.(png|jpe?g|gif|webp)$/i,
+      use: [
+        {
+          loader: "file-loader",
+          options: {
+            name: "/[folder]/[name].[ext]",
+            useRelativePaths: true,
+          },
+        },
+      ],
+    },
+    {
+      test: /\.(woff(2)?|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/,
+      use: [
+        {
+          loader: "file-loader",
+          options: {
+            name: "[name].[ext]",
+            outputPath: "fonts/",
+            publicPath: isOfmPackage ? "fonts/" : "../fonts/",
+            useRelativePaths: true,
+          },
+        },
+      ],
+    },
+  ];
+}
+
+const common_babel_loader_plugins = [
+  "@babel/plugin-proposal-export-default-from",
+  "@babel/plugin-proposal-logical-assignment-operators",
+  ["@babel/plugin-proposal-optional-chaining", { loose: false }],
+  ["@babel/plugin-proposal-pipeline-operator", { proposal: "minimal" }],
+  ["@babel/plugin-proposal-nullish-coalescing-operator", { loose: false }],
+  "@babel/plugin-proposal-do-expressions",
+];
+
+const common_plugins = [
+  new MiniCssExtractPlugin({ filename: "../css/[name].css" }),
+  new CopyWebpackPlugin({ patterns: [{ from: "static/", to: "../" }] }),
+];
+
 module.exports = (env, argv) => {
   // Upload source maps to Sentry for prod builds. Must be the last plugin.
   const appendPlugins =
```

Comment:
Perhaps this is now better described as just "plugins", since it includes everything. The above comment could be relocated to make it clear what it applies to, as well.